oci/service.go

```diff
@@ -934,7 +934,7 @@ func budgetService(ctx context.Context, d *plugin.QueryData, region string) (*se
 // monitoringService returns the service client for OCI Monitoring Service
 func monitoringService(ctx context.Context, d *plugin.QueryData, region string) (*session, error) {
 	logger := plugin.Logger(ctx)
-	serviceCacheKey := fmt.Sprintf("metric explorer-%s", "region")
+	serviceCacheKey := fmt.Sprintf("monitoring-%s", "region")
```

Comment:
@bigdatasourav For the other service we have used the serviceCacheKey as `FunctionsManagement`, `IdentityRegional`.
Should we update it to `Monitoring`?
How we decide the serviceCacheKey currently?